src/docs/reference/scale-to-zero.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Scale to Zero
+---
+
+Scale to Zero means reducing unnecessary costs by ensuring you only use what you need, when you need it.
+
+## What is App Sleeping?
+
+App sleeping enables automated detection of an inactive [service](/develop/services) based on outbound traffic.
+
+Enabling App Sleep on a service tells Railway to stop the service when it is inactive, effectively reducing the overall cost to run it.
```

Comment:
Suggestion: Replace the sentence above ("App sleeping enables automated ...") with the second one ("Enabling App Sleep on a service ..."), so you get a pyramid style of "what it does" -> "how it does it" (i.e. it tells railway to stop the service -> it does this by detecting traffic)